paddle/operators/gru_op.h

```diff
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include "paddle/operators/lstm_op.h"
```

Comment:
Maybe write [`ReorderInitState`](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/lstm_op.h#L28) again in this file, not include `lstm_op.h`.

Reply:
Done.